ui/src/components/page/Page.tsx

```diff
@@ -19,9 +19,11 @@ export const Page: FC<PageProps> = ({ title, footer, children }) => {
   return (
     <Stack
       sx={{
+        '@media (min-width:1000px)': {
```

Comment:
Any reason not to use the [breakpoint helpers](https://mui.com/material-ui/customization/breakpoints/#default-breakpoints) from MUI?